JetSubStructureUtils/Root/BoostedXbbTag.cxx

```diff
@@ -447,13 +489,15 @@ int BoostedXbbTag::result(const xAOD::Jet& jet, std::string algorithm_name, cons
   std::vector<const xAOD::Muon*> matched_muons;
   // first select the muons: Combined, Medium, pT > 10 GeV, |eta| < 2.5
   std::vector<const xAOD::Muon*> preselected_muons(muons->size(), nullptr);
+  // [Next line: selection for muon matching. IF PROBLEMS with getQuality, disable muon matching temporarily by using the second line below instead]
   auto it = std::copy_if(muons->begin(), muons->end(), preselected_muons.begin(), [this](const xAOD::Muon* muon) -> bool { return (muon->pt()/1.e3 > 10.0 && m_muonSelectionTool->getQuality(*muon) <= xAOD::Muon::Medium && fabs(muon->eta()) < 2.5); });
+  //auto it = std::copy_if(muons->begin(), muons->end(), preselected_muons.begin(), [this](const xAOD::Muon* muon) -> bool { return false; });
```

Comment:
This should be set as a configuration in the function rather than having the user comment it out or something.